frontend/app/src/components/home/CurrentUser.svelte

```diff
@@ -23,6 +23,7 @@
     export let user: PartialUserSummary;
 
     function newGroup() {
+        if (client.isReadOnly()) return;
```

Comment:
This seems inconsistent. On the chat summary the menu is not there. Here the message _handler_ is just disabled. I think it's better if you are not allowed to click the menu (either it's not there or it's disabled).